### DISCUSSION TITLE
Triliumserver does not respect the port setting in config.ini, how to change>?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#2745] [Q&A] [answered]
Hi

I want to use it on different port so I changed it in config.ini however it still starts on 8080. Is there any other way that does not involve the environment variable solution? It does not respect the env variable either.

**Comment by sigaloid** [comment]
How are you running it? Docker, server release?

**Comment by gerroon** [comment]
I am running on Debian Testing using , running trilium.sh on the terminal

**Comment by gerroon** [comment]
Nevermind I went back to Docker, it seems easier to change the port

**Comment by zadam** [comment, marked as answer]
FWIW I just tested this and both env. variable configuration and setting config.ini work.
